tests/zenko_tests/node_tests/iam_policies/cloudserver/AssumeRoleWithWebIdentity.js

```diff
@@ -0,0 +1,144 @@
+const assert = require('assert');
+const async = require('async');
+const VaultClient = require('../../VaultClient');
+const { makeGETRequest } = require("../../utils/request");
+const { getS3Client} = require("../../s3SDK");
+const { getSTSClient} = require("../../stsSDK");
+const { getTokenForIdentity } = require("../../utils/getWebIdentityToken");
+
+let iamClient = null;
+let stsClient = null;
+let s3Client = null;
+
+const clientAdmin = VaultClient.getAdminClient();
+const accountName = 'AccountTest';
+const accountInfo = {
+    email: `${accountName}@test.com`,
+    password: 'test',
+};
+const externalAccessKey = 'DZMMJUPWIUK8IWXRP0HQ';
```

Comment:
Is this really hardcoded ?

Reply:
No, the generateAccountAccessKeys is a custom API (superadmin) where we can provide our own access keys. It's noot generated.

Reply:
I don't understand why not just using the ones generated by `generateAccountAccessKeys`

Reply:
I approve the PR because this is not blocking